kolibri/core/discovery/models.py

```diff
@@ -118,8 +124,17 @@ class Meta:
 
     def save(self, *args, **kwargs):
         self.dynamic = False
+        self.is_local = self.capture_network_state()
         return super(StaticNetworkLocation, self).save(*args, **kwargs)
 
+    @classmethod
+    def capture_network_state(self):
+        try:
+            client = NetworkClient.build_from_network_location(self)
+            return ip_address(client.remote_ip).is_private
+        except (AddressValueError, ValueError):
+            return False
```

Comment:
Would you please add this functionality to [this function `capture_network_state`](https://github.com/learningequality/kolibri/blob/develop/kolibri/core/discovery/utils/network/connections.py#L33) instead?